geoportailv3/static/js/search/searchdirective.js

```diff
@@ -340,6 +367,21 @@ app.SearchDirectiveController =
 
 
 /**
+ * @param {Fuse} fuseEngine
+ * @param {string} searchString
+ * @return {Array.<string>}
+ * @private
+ */
+app.SearchDirectiveController.prototype.matchLayers_ =
+    function(fuseEngine, searchString) {
+  var fuseResults = fuseEngine.search(searchString).slice(0, 5);
+  return goog.array.map(fuseResults, function(r) {
+    return r['item'];
```

Comment:
If you correctly type `r`, you should not need to use the bracket notation, as the type of `r` (`FuseResult`) is defined in an externs file.

I'd use something like this:

```js
  var fuseResults = /** @type Array.<FuseResult> */ (fuseEngine.search(searchString).slice(0, 5));
  return goog.array.map(fuseResults,
      /**
       * @param {FuseResult} r
       */
      function(r) {
        return r.item;
      });
```